README.md

```diff
@@ -132,13 +136,14 @@ to master. If you need to create a build locally, you can execute the same
 commands as the CI server.
 
 ```bash
-.travis/build.sh
+yarn build
 ```
 
-## What’s Inside?
+Or
+
+```bash
+.travis/build.sh
+```
 
```

Comment:
to the repo i mean. something like:
"An Ibex over GraphQL/Apollo implementation is available here : githublink